### PR TITLE
Release notes for multi-arch compute machines on  4.13

### PR DESCRIPTION
**Pull request by kelbrown20**
**For Version:** 4.13

**Description:** Cluster with multi-architecture compute machines are now generally available. Highlighting in 4.13 release notes 

**Preview:** 
[Release notes -> Post-installation configuration ](https://58844--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-post-installation)

**QE review:**
- [x] QE approval

### CODE REVIEW

**Comment by kelbrown20** [comment]
@Prashanth684 Hey Prashanth! Can you take a look at these releaese notes when you get the chance? Thank you!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -154,6 +154,12 @@ If you need the virtual IP (VIP) addresses to run on the control plane nodes in
 [id="ocp-4-13-post-installation"]
 === Post-installation configuration
 
+[id="ocp-4-13-multi-arch-compute-machines-ga"]
+==== {product-title} clusters with multi-architecture compute machines 
+
+{product-title} {product-version} clusters with multi-architecture compute machines is now generally available. You can now create a cluster with compute nodes of different architectures on AWS and Azure installer provisioned infrastructures. Bare metal user provisioned installations are in Technology Preview. For more information on creating a cluster with multi-architecture compute machines, see xref:../post_installation_configuration/multi-architecture-configuration.adoc#multi-architecture-configuration[Configuring multi-architecture compute machines on an OpenShift Container Platform cluster].
```
Comment (Prashanth684) [inline release_notes/ocp-4-13-release-notes.adoc:160]: lgtm ....

```
You can now create a cluster with compute nodes of different architectures on AWS and Azure installer provisioned infrastructures
````
yes we do provision with IPI but the mixed arch compute nodes are added as a day2 operation right? is there a succinct way to mention this?

**Comment by Prashanth684** [comment]
lgtm

**Review by abhatt-rh — COMMENTED** [review]
Hi @kelbrown20,
Mostly LGTM with some minor comments

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by kelbrown20** [comment]
@aleskandro Hey Alessandro! Can you also take a look at this release note when you get the chance? Thank you!

**Comment by aleskandro** [comment]
Hi @kelbrown20 thanks!

/label qe-approved

**Pull request merged**